backend/plugins/frontline_api/src/modules/integrations/imap/utils.ts

```
@@ -1,502 +1,481 @@
+          data: {
+            _id: integration._id as string,
+          },
+        });
+      }
+    } catch (e) {
+      await lock.unlock();
+    }
+  };
+  // wait for other containers to start up
+  NODE_ENV === 'production' &&
+    (await new Promise((resolve) => setTimeout(resolve, 60000)));
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)